ea_restaurant_data_loader/lib/service/app_client_service.ex

```diff
@@ -0,0 +1,10 @@
+defmodule EaRestaurantDataLoader.AppClientService do
+    
+    alias EaRestaurantDataLoader.Repo 
+    alias EaRestaurantDataLoader.AppClient
+    import Ecto.Query
+
+    def get_client_by_client_id(client_id) do
+        Repo.all(from(ac in AppClient, where: ac.client_id == ^client_id))
```

Comment:
You can remove parenthesis to make it look more fluent:
```
Repo.all(from ac in AppClient, where: ac.client_id == ^client_id)
```

Reply:
Done.